test/test_lib/utils.js

```diff
@@ -335,6 +335,12 @@ Utils.prototype = {
     '0x0000000000000000000000000000000000000000',
 
   NULL_ADDRESS: '0x0000000000000000000000000000000000000000',
+
+  SENTINEL_ADDRESS: '0x0000000000000000000000000000000000000001',
+
+  MAX_UINT256: new BN(
+    '115792089237316195423570985008687907853269984665640564039457584007913129639935'
```

Comment:
I am not sure, but assuming new BN('0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff', 16) should work ...